ew/static/cfg.py

```diff
@@ -41,6 +41,7 @@
 #sure, but is slimernalia active?
 slimernalia_active = True
 
+forbidden_channels = ["suggestion-box", "splatify"]
```

Comment:
Splatify is actually a subzone in crookline too, so we can't prevent command responses there. Suggestion box isn't either, but whether you leave it in is up to you